static-analysis/autodispose-lint/src/test/kotlin/com/uber/autodispose/lint/LintFilesExtension.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2018. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.lint
+
+import com.android.tools.lint.checks.infrastructure.TestFiles.bytes
+
+fun Any.rxJava2() = bytes("libs/rxjava-2.1.7.jar", javaClass.getResourceAsStream("/rxjava-2.1.7.jar").readBytes())
```

Comment:
This is clever, but I also wonder if there isn't maybe a better way for this, like yanking it from a gradle dependency. Not important to figure out for this PR though